imports/ui/pages/root-redirector.js

```diff
@@ -6,16 +6,18 @@ import { FlowRouter } from 'meteor/kadira:flow-router';
 import './root-redirector.html';
 
 Template.app_rootRedirector.onCreated(() => {
-  // We need to set a timeout here so that we don't redirect from inside a redirection which is a no-no in FlowRouter.
-  Meteor.setTimeout(() => {
+  Template.instance().autorun(() => {
+    Template.instance().subscribe('memberships.ofUser', { userId: Meteor.userId() });
```

Comment:
Vajon mi tortenik, ha nincs userId, es feliratkozunk az undefined user membershipjeire? Nem szall el a szerver?
Nem lehet jobb lenne a subscription az (if Meteor.userId())-n belul?

Reply:
Kattintós tesztnél ilyen nem történt, de tényleg úgy korrekt, ha (if Meteor.userId())-n belül van. Viszont mintha lassabban töltene be az oldal ezzel az extra feliratkozással, mint eddig, de lehet, hogy ez csak illúzió.